unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/DebugParameters/DrawIfAttribute.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
+using System;
+ 
+/// <summary>
+/// Draws the field/property ONLY if the compared property compared by the comparison type with the value of comparedValue returns true.
+/// Based on: https://forum.unity.com/threads/draw-a-field-only-if-a-condition-is-met.448855/
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = true)]
+public class DrawIfAttribute : PropertyAttribute
```

Comment:
Perhaps we could move this class to a more general location. We can use it in other scripts, as more editor utils. We have already `ExpandableAttribute.cs`, however, it seems that it's not used, we could make a shared assembly for those.

Reply:
Makes sense. I'll make a shared assembly between this class and `ExpandableAttributes.cs`, so all the editor utils are centralized in one place.